### PR TITLE
fixed error on no tracks

### PR DESCRIPTION
**Pull request by yofukashino**
was giving errors when no tracks so fixed that shit senpai

### CODE REVIEW

File: src/commands/play.js [hunk 1]
```diff
@@ -34,11 +34,11 @@ module.exports = {
         });
 
         try {
-            const { type, playlistName, tracks } = await music.load(util.isValidURL(query) ? query : `ytsearch:${query}`);
-            if (!tracks.length) return ctx.respond({
+               const result =  await music.load(util.isValidURL(query) ? query : `ytsearch:${query}`);
```
Comment (Allvaa) [inline src/commands/play.js]:
```suggestion
            const result =  await music.load(util.isValidURL(query) ? query : `ytsearch:${query}`);
```

File: src/commands/play.js [hunk 2]
```diff
@@ -34,11 +34,11 @@ module.exports = {
...
+            if (!result || !result.tracks.length)  return ctx.respond({
```
Comment (Allvaa) [inline src/commands/play.js]:
```suggestion
            if (!result?.tracks.length)  return ctx.respond({
```
use optional chaining

File: src/commands/play.js [hunk 3]
```diff
@@ -34,11 +34,11 @@ module.exports = {
...
                 embeds: [util.embed().setDescription("❌ | Couldn't find any results.")]
             });
-            
+            const { type, playlistName, tracks } = result;
```
Comment (Allvaa) [inline src/commands/play.js]:
```suggestion
            const { type, playlistName, tracks } = result;
```

**Review by Allvaa — CHANGES_REQUESTED** [review]

**Comment by yofukashino** [comment]
well did the changes :)

**Review by Allvaa — APPROVED** [review]
ok looks good, thanks

**Pull request merged**